### ISSUE TITLE
In Unified Interface, how do I click a button within a JavaScript confirmation dialog?

### ISSUE THREAD

**Issue by simonbridgwater** [Microsoft/EasyRepro#673]
I have a Unified Interface application containing lots of command bar buttons which call various JavaScript functions. Many of these functions use the `confirm()` method to get confirmation from the user that they wish to proceed, before continuing with the rest of the function.

I'm using `XrmApp.CommandBar.ClickCommand()` to click the command bar button. The confirmation dialog does appear very briefly, but almost immediately disappears again. I'm unsure how to overcome this, as even if there is an EasyRepro solution to clicking the 'OK' button in the confirmation dialog, that line of code won't be executed until after the dialog has already disappeared.

Has anyone else encountered or overcome this problem?

If it matters, my tests are carried out using Chrome.

**Comment by TheSarcasticGuy** [comment]
I'm facing the issue. Have anyone found any workaround for this?

**Comment by thathi** [comment]
@TheSarcasticGuy @simonbridgwater xrmApp.Dialogs.ConfirmationDialog(true); Is this the one you are looking?

**Comment by TheSarcasticGuy** [comment]
> @TheSarcasticGuy @simonbridgwater xrmApp.Dialogs.ConfirmationDialog(true); Is this the one you are looking?

Nope. Actually the issue is after using xrmApp.CommandBar.ClickCommand("custom command button name"), a confirmation alert is shown. Further code is executed after clicking on Ok button from the alert box. While doing this through EasyRepro, the alert disappears in fraction of seconds. I was wondering how to interact with that alert box?
